Dockerfile

```diff
@@ -5,3 +5,11 @@ RUN pip install numpy pandas flask pygal smalluuid zipstream python-dateutil git
 WORKDIR /app
 
 ADD . /app
+
+RUN pip install -r dev-requirements.txt
```

Comment:
Can we remove line 3 here?

Reply:
Ah yeah, that's not needed anymore.  Removed.